resources/innosetup/setup.iss

```diff
@@ -57,15 +57,31 @@ Name: "webdav"; Description: "WebDAV system configuration"; Types: full; ExtraDi
 
 [Registry]
 Root: HKLM; Subkey: "SYSTEM\CurrentControlSet\Services\WebClient\Parameters"; ValueType: dword; ValueName: "FileSizeLimitInBytes"; ValueData: "$ffffffff"; Components: webdav
-Root: HKCR; Subkey: ".cryptomator"; ValueType: string;  ValueName: ""; ValueData: "CryptomatorMasterkeyFile"; Flags: uninsdeletekey
-Root: HKCR; Subkey: "CryptomatorMasterkeyFile"; ValueType: string;  ValueName: ""; ValueData: "Cryptomator Masterkey File"; Flags: uninsdeletekey
-Root: HKCR; Subkey: "CryptomatorMasterkeyFile\DefaultIcon"; ValueType: string; ValueName: ""; ValueData: "{app}\Cryptomator.exe,0"
-Root: HKCR; Subkey: "CryptomatorMasterkeyFile\shell\open\command"; ValueType: string; ValueName: ""; ValueData: """{app}\Cryptomator.exe"" ""%1"""
-Root: HKCR; Subkey: ".c9r"; ValueType: string;  ValueName: ""; ValueData: "EncryptedCryptomatorVaultFile"; Flags: uninsdeletekey
-Root: HKCR; Subkey: "EncryptedCryptomatorVaultFile"; ValueType: string;  ValueName: ""; ValueData: "Encrypted Cryptomator vault file"; Flags: uninsdeletekey
-Root: HKCR; Subkey: "EncryptedCryptomatorVaultFile\DefaultIcon"; ValueType: string; ValueName: ""; ValueData: "{app}\Cryptomator.exe,0"
+
+; Remove legacy AutoStart entry; see: https://github.com/cryptomator/cryptomator-win/issues/33
 Root: HKCU; Subkey: "SOFTWARE\Microsoft\Windows\CurrentVersion\Run\"; ValueType: none; ValueName: "Cryptomator"; Flags: dontcreatekey deletevalue
 
+; .cryptomator filetype: Add extension, MIME type, perceived type, description, icon and command
+Root: HKA; Subkey: "Software\Classes\.cryptomator"; ValueType: string;  ValueName: ""; ValueData: "CryptomatorMasterkeyFile"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\.cryptomator"; ValueType: string; ValueName: "Content Type"; ValueData: "application/x-vnd.cryptomator.vault-metadata"
+Root: HKA; Subkey: "Software\Classes\.cryptomator"; ValueType: string; ValueName: "PerceivedType"; ValueData: "text"
+
+Root: HKA; Subkey: "Software\Classes\CryptomatorMasterkeyFile"; ValueType: string;  ValueName: ""; ValueData: "Cryptomator Masterkey File"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\CryptomatorMasterkeyFile\DefaultIcon"; ValueType: string; ValueName: ""; ValueData: "{app}\Cryptomator.exe,0"
+Root: HKA; Subkey: "Software\Classes\CryptomatorMasterkeyFile\shell\open\command"; ValueType: string; ValueName: ""; ValueData: """{app}\Cryptomator.exe"" ""%1"""
+
+; .c9r & .c9s filetypes: Add extension, MIME type, perceived type, description and icon for each
+Root: HKA; Subkey: "Software\Classes\.c9r"; ValueType: string;  ValueName: ""; ValueData: "CryptomatorEncryptedData"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\.c9r"; ValueType: string; ValueName: "Content Type"; ValueData: "application/x-vnd.cryptomator.encrypted-data"
+Root: HKA; Subkey: "Software\Classes\.c9r"; ValueType: string; ValueName: "PerceivedType"; ValueData: "system"
```

Comment:
What other options do we have for `PerceivedType`? Is there something like `binary`?

Reply:
I don't know about custom types, but currently that's all there is:
> The default perceived types are as follows:
- Folder
- Text
- Image
- Audio
- Video
- Compressed
- Document
- System
- Application
- Gamemedia
- Contacts

See: [Here](https://web.archive.org/web/20200410143821/https://docs.microsoft.com/en-us/windows/win32/shell/fa-perceivedtypes)

Reply:
Not exactly a lot of choice, here. Entropywise, `Compressed` would come closest, but would be very misleading if 3rd party applications start feeling the urge to "unzip" a c9r file.

Reply:
Correct. And that's exactly what would happen: The user would receive the suggestion to use a ZIP Application to open the file.